src/Helpers/Utility/TraceId.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Pantheon\Terminus\Helpers\Utility;
+
+class TraceId
+{
+    private static $traceId;
+
+    /**
+     * Generate UUID for use as distributed tracing ID and assign to static class variable
+     */
+    public static function generateTraceId()
```

Comment:
This is good. We also need to make Request.php use this utility class, so that the trace id sent to ygg through hermes/papi is the same id that goes to pantheon sshd.

Reply:
nod.. it looks like that's being done..